### ISSUE TITLE
Can't build master on Mac OS X 10.7.5

### ISSUE THREAD

**Issue by simpsora** [ntop/ntopng#90]
After running `autogen.sh`, `configure.sh`, and the same for nDPI master, I get the following error during `make`:

```
$ make
/usr/bin/clang++ -g -Wall -I/Users/rsimpson/code/ntopng -I/Users/rsimpson/code/ntopng/include -I/usr/local/include -I /Users/rsimpson/code/ntopng/third-party/hiredis -I/Users/rsimpson/code/ntopng/third-party/mongoose -I/usr/local/Cellar/json-c/0.12/include/json-c  -I/usr/local/Cellar/ndpi/1.6/include/libndpi-1.6.0/libndpi  -I/usr/local/Cellar/luajit/2.0.4/include/luajit-2.0  -I/usr/local/Cellar/rrdtool/1.5.3/include  -I/usr/local/Cellar/zeromq/4.1.2/include    -I/Users/rsimpson/code/ntopng -I/Users/rsimpson/code/ntopng/include -I/usr/local/include -I/Users/rsimpson/code/ntopng/third-party/http-client-c/src/ -I/Users/rsimpson/code/ntopng/third-party/EWAHBoolArray/headers  -DDATA_DIR='"/usr/local/share"' -I/Users/rsimpson/code/ntopng/third-party/libgeohash -I/Users/rsimpson/code/ntopng/third-party/patricia   -c src/ActivityStats.cpp -o src/ActivityStats.o
In file included from src/ActivityStats.cpp:22:
In file included from /Users/rsimpson/code/ntopng/include/ntop_includes.h:180:
/Users/rsimpson/code/ntopng/include/Host.h:154:23: error: unknown type name 'ndpi_protocol'
  bool doDropProtocol(ndpi_protocol l7_proto);
                      ^
In file included from src/ActivityStats.cpp:22:
In file included from /Users/rsimpson/code/ntopng/include/ntop_includes.h:181:
/Users/rsimpson/code/ntopng/include/Flow.h:43:3: error: unknown type name 'ndpi_protocol'
  ndpi_protocol ndpi_detected_protocol;
  ^
/Users/rsimpson/code/ntopng/include/Flow.h:159:28: error: unknown type name 'ndpi_protocol'
  void setDetectedProtocol(ndpi_protocol proto_id);
                           ^
/Users/rsimpson/code/ntopng/include/Flow.h:179:10: error: unknown type name 'ndpi_protocol'
  inline ndpi_protocol get_detected_protocol()    { return(ndpi_detected_protocol);          };
         ^
4 errors generated.
make: *** [src/ActivityStats.o] Error 1
```

The version in brew doesn't work for me (keep getting `22/Jul/2015 00:07:12 [src/Lua.cpp:52] ERROR: Unable to create Lua interpreter`), so I was trying to build from master.

Any suggestions?

**Comment by ariava** [comment]
@simpsora are you using the latest nDPI? Which branch?

**Comment by lucaderi** [comment]
You are using the wrong DPI. If you need the devel version of ntopng, you need to use the devel version of nDPI and the other way round. Please fix this and it will work, otherwise wait until we update brew.

**Comment by simpsora** [comment]
Hi @ariava and @lucaderi,

I am building using the master branch of both projects.  I don't need the devel version, just a standard version to use as a user.

I cloned the master branch of `ntopng`, and when I ran `configure` got this output:

```
checking for nDPI... not found
Please do cd ..; git clone https://github.com/ntop/nDPI.git; cd nDPI; ./autogen.sh; make; cd ../ntopng
 and try again
```

I did that (it built cleanly), returned to `ntopng` and got the make errors described above.

What's the "right" DPI, and where would I find it?  I didn't see anything relevant in `README.ntopng` nor in the user guide.

Thanks again!
